src/biokbase/narrative/tests/narrative_mock/mockclients.py

```diff
@@ -76,33 +82,20 @@ def test_my_function(self):
     """
 
     config = ConfigTests()
-    _ee2_job_info = config.load_json_file(config.get("jobs", "ee2_job_info_file"))
+    _job_state_data = TEST_JOBS
```

Comment:
calling the variable `_ee2_job_info` was confusing as the job info command does something completely different => renamed to `job_state_data`